package.json

```diff
@@ -52,7 +52,7 @@
     "@types/tmp": "0.2.6",
     "codecov": "^3.0.0",
     "deep-copy": "^1.4.2",
-    "gts": "^5.0.0",
+    "gts": "^5.3.1",
```

Comment:
LGTM to fix Kokoro

But discussed offline and the underlying issue is that the NPM version coming with Node 14 does not support the lockfile version 3 and so npm is pulling in different versions than in `package-lock.json`. This should be fixed so the build is hermetic

Reply:
https://github.com/google/pprof-nodejs/pull/298

@psx95 can you take a look after you merge this PR?

Reply:
Sure, I will take a look